arches/app/utils/data_management/resources/formats/rdffile.py

```diff
@@ -650,6 +650,13 @@ def find_matching_branch(self, k, v, tree_node, result, tile=None, indent=0):
                         else:
                             self.printline(f"Could not validate {values} as a {o['datatype']}", indent + 1)
                     else:
+                        if o["datatype"].datatype_name == 'date':
+                            try:
```

Comment:
```suggestion
                        if self.default_timezone and o["datatype"].datatype_name == 'date':
```

Reply:
Won't this fail if the value can't be parsed on this line

`datetime.datetime.strptime(value, "%Y-%m-%d %H:%M:%S")`

Reply:
sorry meant my suggestion to be this:
```suggestion
                        if self.default_timezone and o["datatype"].datatype_name == 'date':
                            try:
```